ui/helpers/constants/settings.js

```diff
@@ -0,0 +1,342 @@
+import {
+  ALERTS_ROUTE,
+  ADVANCED_ROUTE,
+  SECURITY_ROUTE,
+  GENERAL_ROUTE,
+  ABOUT_US_ROUTE,
+  NETWORKS_ROUTE,
+  CONTACT_LIST_ROUTE,
+  EXPERIMENTAL_ROUTE,
+  ///: BEGIN:ONLY_INCLUDE_IN(flask)
+  SNAPS_LIST_ROUTE,
+  ///: END:ONLY_INCLUDE_IN
+} from './routes';
+
+export const SETTINGS_ROUTES = [
```

Comment:
These aren't just routes. Maybe something like SETTINGS_CONSTANTS would be better?

Reply:
sure